sql/driver/conn.go

```diff
@@ -78,24 +88,23 @@ func (c *conn) Query(query string, args []driver.Value) (driver.Rows, error) {
 	if err != nil {
 		return nil, err
 	}
-	return c.query(stmt, args)
+	return c.query(createCall(query, args), stmt, args)
```

Comment:
Rather than creating the call here, I would have just created it inside `conn.query` for those statements that aren't being handled locally. The parsed query can be transformed back into SQL using `parser.Statement.String()`. Not a big deal as this will likely see reworking again when we move everything to the server.